### ISSUE TITLE
The performance of

### ISSUE THREAD

**Issue by linyijun0406** [INVOKERer/DeepRFT#24]

**Comment by linyijun0406** [comment]
The extraction code of Baidu Netdisk is not provided. We downloaded and tested the pre-trained model from Google Drive, but it doesn't seem to work for image deblurring/defocusing. Besides that, we have tried to train the network on the GoPro dataset, but the loss converged very slowly.

**Comment by INVOKERer** [comment]
Do you use the test code we provide?

**Comment by INVOKERer** [comment]
You can get PSNR directly by the new test.py. If it still doesn't work, please let me know. Thank you.

**Comment by linyijun0406** [comment]
The pre-trained model can be used normally for deblurring. Thanks very much.

**Comment by INVOKERer** [comment]
So, the link of Baidu Netdisk doesn't work by clicking it?

**Comment by linyijun0406** [comment]
> So, the link of Baidu Netdisk doesn't work by clicking it?
It requires the extraction code.
<img width="555" alt="image" src="https://user-images.githubusercontent.com/30616901/163079845-9b925f44-5034-4ac9-9ee7-90f161e32a41.png">

**Comment by INVOKERer** [comment]
Sorry, I forgot it.
